packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/index.jsx

```diff
@@ -87,6 +93,18 @@ const ConfigurationTab = ({
 
   const hasLoginError = error && error.isLoginError()
 
+  const handleDeleteConfirm = async () => {
+    setRequestDeletion(false)
+    const extensionInstalled = await CozyUtils.checkHasInstalledExtension(
```

Comment:
If the user is on a device that does not have the extension installed, the cipher should be unshared right ?

Reply:
Yes, but `checkHasInstalledExtension` does not says if the device has the extension, it says if it's been installed once, and so if the vault is set up.

Reply:
Shouldn't the condition be along the line of `hasVault && vault.isLocked == true` ?

Reply:
OK, weird naming then :/ The vault can already be unlocked right, can we check if it's unlocked ?

Reply:
Yes the naming is definitely not ideal, it's because it's based on a field `extensionInstalled` in the settings, that was originally used for the Cozy Pass web app onboarding, but was later recycled to know if the vault was used or not. 

> The vault can already be unlocked right, can we check if it's unlocked 

This is actually a subject we're hoping to address for next year, but it's probably out of scope here, I understood it has security implications

Reply:
OK fine, thanks.